tests/SlimFaas.Tests/HistorySynchronizationWorkerShould.cs

```diff
@@ -12,19 +13,17 @@ public async Task SyncLastTicksBetweenDatabaseAndMemory()
         var redisMockService = new RedisMockService();
         var historyHttpRedisService = new HistoryHttpRedisService(redisMockService);
         var kubernetesService = new Mock<IKubernetesService>();
-        var deploymentsInformations = new DeploymentsInformations()
+        var deploymentsInformations = new DeploymentsInformations(Functions: new List<DeploymentInformation>()
```

Comment:
it looks weird to pass an empty list here, i think DeploymentsInformations should:
- be renamed DeploymentInformation
- initialize its own porperties internally